sympy/utilities/tests/test_lambdify.py

```diff
@@ -460,6 +469,24 @@ def test_numexpr_userfunctions():
     func = lambdify((x, y), uf(x, y), modules='numexpr')
     assert numpy.allclose(func(a, b), 2*a*b+1)
 
+def test_tensorflow_basic_math():
+    if not tensorflow:
+        skip("tensorflow not installed.")
+    expr = Max(sin(x), Abs(1/(x+2)))
+    func = lambdify(x, expr, modules="tensorflow")
+    a = tensorflow.constant(0, dtype=tensorflow.float32)
+    s =  tensorflow.Session()
+    assert func(a).eval(session=s) == 0.5
+
+def test_tensorflow_placeholders():
+    if not tensorflow:
+        skip("tensorflow not installed.")
```

Comment:
My change seems to break the build.

One test failed with `/home/travis/build.sh: line 45: /home/travis/virtualenv/python3.5_with_system_site_packages/bin/activate: No such file or directory`

Another failed with a syntax error at `/home/travis/build.sh: line 265: `if [[ ! -f ~/virtualenv/python[2.7, 3.3, 3.4, 3.5]/bin/activate ]]; then'`

Should I put the dummy value back in? This is my first time dealing with Travis, apologies for being dense.